examples/03-single_node-haproxy/docker-compose (secrets).yml

```diff
@@ -0,0 +1,142 @@
+version: '3.5'
```

Comment:
Moved the secrets example to its own file (secrets added too much clutter and detracted from legibility)